packages/E2ETest/package.json

```diff
@@ -30,16 +30,14 @@
     "@types/react": "16.8.15",
     "@types/react-native": "~0.57.51",
     "@wdio/appium-service": "^5.12.1",
-    "@types/chai": "^4.1.7",
-    "@types/mocha": "^5.2.7",
+    "@types/jasmine": "2.8.7",
```

Comment:
We may as well use ^ types here, so that I pick up minor updates as they are released.

Reply:
Done